x/ibc/02-client/types/tendermint/misbehaviour.go

```diff
@@ -3,10 +3,67 @@ package tendermint
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types/errors"
+
+	"github.com/tendermint/tendermint/crypto"
+	tmtypes "github.com/tendermint/tendermint/types"
 )
 
+// Evidence is a wrapper over tendermint's DuplicateVoteEvidence
+// that implements Evidence interface expected by ICS-02
+type Evidence struct {
+	tmtypes.DuplicateVoteEvidence
+	ChainID        string
+	ValPubKey      crypto.PubKey
+	ValidatorPower int64
+	TotalPower     int64
+}
+
+// Type implements exported.Evidence interface
+func (ev Evidence) Route() string {
+	return exported.ClientTypeTendermint
+}
+
+// Type implements exported.Evidence interface
+func (ev Evidence) Type() string {
+	return exported.ClientTypeTendermint
+}
+
+// String implements exported.Evidence interface
+func (ev Evidence) String() string {
+	return ev.DuplicateVoteEvidence.String()
```

Comment:
I think we can add the other fields here as well. We can implement the `String` function by using marshal yaml to bytes and then print them.  We'd need to write a test for the expected string tho.